src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4078,7 +4078,9 @@ internal static string GetClearHostFunctionText()
             {
                 // Porting note: non-Windows platforms use `clear`
                 return @"
-& (Get-Command -CommandType Application clear | Select-Object -First 1).Definition
+[Console]::Write((
```

Comment:
Should we use `Write-Host` here instead?  There may be cases where there isn't a console?

Reply:
@SteveL-MSFT: In principle, `$host.UI.Write()` is better than `[Console]::Write()`, but note that we're calling a _terminal-specific_  utility anyway - see also: https://github.com/PowerShell/PowerShell/issues/10181#issuecomment-537510875

Reply:
@iSazonov can you validate if this works remotely?

Reply:
@SteveL-MSFT: No, it doesn't (see linked comment), but it also didn't work before. However, if we're willing to assume an `xterm` terminal (which sounds like a safe bet), we can fix it as follows before calling `clear`: `if (-not $env:TERM) { $env:TERM='xterm' }` - probably better to reset the variable afterwards.

Reply:
Ok, we can defer the remote scenario since it doesn't work today (and I think there's a issue on that already).  We can take this with the current change to address the specific problem of the VT codes emitted to the pipeline.

Reply:
@SteveL-MSFT: Couldn't find an issue relating to remoting specifically.

As it turns out, my suggested `$env:TERM='xterm'` workaround for remoting only works with `$host.UI.Write()` (`Write-Host -NoNewline`), not with `[console]::Write()`, because the escape sequences must be sent to the _caller's_ terminal.

In light of that - if you don't want to incorporate the full workaround yet - it's better to switch to `$host.UI.Write()`.

For the record, here's the complete workaround (from my personal notes):

---

To fix the remoting problem, `$env:TERM` can be set to `xterm` temporarily, under the assumption that pretty much all terminals are Xterm-compatible these days - and that it is the _caller_ that uses such a terminal, because the escape sequences must be printed to the _caller's_ terminal:

```powershell
  function Clear-Host {
    if ($noTerm = -not $env:TERM) { $env:TERM = 'xterm' }
    $host.UI.Write((
      & (Get-Command -CommandType Application clear | Select-Object -First 1).Definition
    ))
    if ($noTerm) { $env:TERM = $null }
  }
```

Reply:
I like the suggestion to switch to `$host.UI.Write()`.  Not so sure about the assumption that terminals are xterm compatible.  The issue I was thinking about is https://github.com/PowerShell/PowerShell/issues/8606 where remoting was discussed but wasn't the focus.

Reply:
I will look `$host.UI.Write()` today. I hope we could implement this step-by-step (first for local host. then remote one that would be more tricky)

Reply:
Thanks for the link, @SteveL-MSFT, but resolving #8608 will not address the remoting problem; while there is a single comment - https://github.com/PowerShell/PowerShell/issues/8606#issuecomment-452435884 - that touches on the need to support non-console hosts, a separate issue should really be opened for that.

As for assuming `xterm`: Anecdotally, I'd say it's a pretty safe bet, but the better solution would be to extend the `$PSSenderInfo` object to include the caller's `$env:TERM` value, if any, and have `Clear-Host` consult that.